pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -98,12 +105,38 @@ findDependency() {
     return 1
 }
 
+patchElfInterpreter() {
+    local toPatch=$1
+    local linkers=( \
+        "$NIX_CC/nix-support/dynamic-linker" \
```

Comment:
You don't need to use backslashes for continuations here, besides, why have `autoPatchelfLinkers` *and* `linkers` here again?